code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -106,10 +106,10 @@
 	name = "Fireball"
 	spell_type = /obj/effect/proc_holder/spell/aimed/fireball
 
+/*
 /datum/spellbook_entry/rod_form
 	name = "Rod Form"
-	spell_type = /obj/effect/proc_holder/spell/targeted/rod_form
-	cost = 40
+	spell_type = /obj/effect/proc_holder/spell/targeted/rod_form */
```

Comment:
Magic missile - tracking stun x4, unblockable
Disintegrate - instant gib, unblockable
Fireball - AoE Fire/explosion, targeted, unblockable. Use in a 3x3 area to win
Lighting bolt - Electrical stun, targeted, unblockable, Use in an ambush situation for instant winning.
Staff of chaos - Instant death, Fireball, or polymorph has options. All unblockable
Wands - Firebolt/death/change
Hulk mode - Laser eyes and immunity to stuns, punches send people flying and stun them
The Traps - multiple damaging traps + a stun trap, easily dropped if running.
Time stop - complete projectile protection/avoidance. 

If your entire argument is about how much you enjoy the rodform as the only reliable way to be robust, I dunno what to tell you fam. anything that instantly puts you in a helpless state is literal death in this game.